### ISSUE TITLE
Make the drone move in a straight line

### ISSUE THREAD

**Issue by kayakaan02** [dji-sdk/Mobile-SDK-Android#1244]

**Comment by dji-dev** [comment]
Agent comment from Hummels Lei in <a href='https://djisdksupport.zendesk.com/agent/tickets/90179'>Zendesk ticket #90179</a>:<br><br>In order to fulfill your request, may I kindly ask you to utilize the virtual joystick to navigate the aircraft and maintain a straight flight path?
Please let me know if you have any further questions or if there is anything else I can assist you with.
<br><p><em><small>°°°</small></em></p>

**Comment by kayakaan02** [comment]
> Agent comment from Hummels Lei in [Zendesk ticket #90179](https://djisdksupport.zendesk.com/agent/tickets/90179):In order to fulfill your request, may I kindly ask you to utilize the virtual joystick to navigate the aircraft and maintain a straight flight path? Please let me know if you have any further questions or if there is anything else I can assist you with.
> 
> _°°°_

Yes I can do that but sometimes the wind can be strong or when the drone stands still it can move to other directions on its own, how can I make sure that that doesn't happen and if it happens how can I be notified of this situation?
